server/api.js

```diff
@@ -1,9 +1,10 @@
 import { Router } from "express";
+import { query } from "./db";
 
 const router = Router();
 
 router.get("/", (_, res) => {
-	res.status(200).json({ message: "WELCOME TO LOVE ME TENDER SITE" });
+	res.status(200).json({ message: "TENDER MANAGEMENT SYSTEM" });
```

Comment:
Remove this change. It's unrelated to the commit and probably unnecessary

Reply:
I have changed it